### DISCUSSION TITLE
記事の編集をセクション単位でも可能にしてほしい

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#113] [要望・アイデア・不具合]
公開した記事を再度編集する場合や、編集リクエストを送る場合など、セクション単位に編集モードに入れると便利なケースがあると思います。

特に、長文の記事の途中で誤字を見つけた場合などに、誤字を見つけたセクションだけの編集モードにできると、すぐに修正ができて便利だと思います。

すぐには難しそうだなと思いますが、ご一考いただけるとありがたいです。

**Comment by kskwtnk** [comment]
@yoshi389111 
ご意見ありがとうございます！

長文記事の編集、たしかに大変ですよね……私自身も感じる場面は結構あります 🙇 

すぐには実現できなさそうなのですが、編集リクエストでの体験も含めてもっと良くしていきたいポイントなので、今後の参考にさせていただきます！ありがとうございます！

**Comment by munierujp** [comment]
最近よくWIkipediaを編集しているのですが、Wikipediaの場合はセクション単位での編集が可能で、更新内容にも自動的にセクション名が記載されるので、長い記事でも編集しやすいと思いました。
Qiitaでも同様の機能があるといいですね。

例：

<img width="1114" alt="スクリーンショット 2022-10-08 1 16 39" src="https://user-images.githubusercontent.com/20086673/194600429-0395095e-ef1e-4208-b64f-d2d96ae5b559.png">

**Reply by mkdk09** [reply]
@munierujp
アイデアありがとうございます！
